### ISSUE TITLE
Aria labels for elements with roles: checkbox, radio, switch, and button

### ISSUE THREAD

**Issue by selenatuyen** [primefaces/primeng#9019]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[ x ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
After #8439 was introduced, input/toggle fields with roles set equal to checkbox, radio, switch, and button expect an accessible name/label. And role set equal to button expect discernible text. There are no aria-label or aria-labelledby attributes available on most of these fields that had roles added to them. Since these labels are not available it violates accessibility standards.

**Expected behavior**
Add aria-label and/or aria-labelledby attributes to the divs that had roles of checkbox, radio, switch, and button added to them in #8439

**Minimal reproduction of the problem with instructions**
Example: in dropdown.ts  role="button" was added to this div, I would've expected an aria-label attribute to label it as well.
```
<div class="ui-dropdown-trigger ui-state-default ui-corner-right" role="button" aria-haspopup="listbox" [attr.aria-expanded]="overlayVisible">
       <span class="ui-dropdown-trigger-icon ui-clickable" [ngClass]="dropdownIcon"></span>
 </div>
```

**What is the motivation / use case for changing the behavior?**
Meeting accessibility requirements.
[https://web.dev/aria-toggle-field-name/](https://web.dev/aria-toggle-field-name/)
[https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name](https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name)
[https://dequeuniversity.com/rules/axe/3.3/button-name](https://dequeuniversity.com/rules/axe/3.3/button-name)

**Please tell us about your environment:**
Windows

* **Angular version:** 8.2.14

* **PrimeNG version:** 9.0.6

* **Browser:** Chrome 83
 
* **Language:**  TypeScript 3.5.3

* **Node (for AoT issues):** `node --version` =   v12.8.0

**Comment by kkdattaGIT** [comment]
@yigitfindikli / @cagataycivici wondering if you have looked at the issue raised by SelenaTuyen?
We have critical bugs for Accessibility Tests failing for the limitation mentioned above.

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
